Week1-Coding-Task.hs

```diff
@@ -0,0 +1,76 @@
+--Section 1, credit card validation
+--Double value of every second digit from beginning of right 
+--Add digits of doubled and undoubled values to form new number 
+--Calculate result mod 10: if 0, valid 
+
+------------------------------------------------------------------------------
+--Exercise 1: toDigits and toDigitsRev 
+------------------------------------------------------------------------------
+toDigits :: Integer -> [Integer]
```

Comment:
Usually types should go immediately before implementation:

```haskell
toDigits :: Integer -> [Integer]
toDigits s = ...
```

This helps readability a lot.